### PR TITLE
Paulo Sergio - Feed API Challenge

### PR DESCRIPTION
**Pull request by ps1312**

### CODE REVIEW

**Comment by mapostolakis** [comment]
Hi @ps1312, please fix the name of the PR following the instructions in the README. Thank you!

File: FeedAPIChallenge/Feed Feature/FeedImage.swift [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import Foundation
 
-public struct FeedImage: Hashable {
+public struct FeedImage: Hashable, Decodable {
```
Comment (mapostolakis) [inline FeedAPIChallenge/Feed Feature/FeedImage.swift:7]: You can revert this change. The `Decodable` conformance is not required.

**Review by mapostolakis — COMMENTED** [review]
Hi @ps1312,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs.

I left one comment regarding a protocol conformance that can be omitted. Please, feel free to push a new commit.

Great job! ✅

File: FeedAPIChallenge/Feed Feature/FeedImage.swift (hunk 1 above)
Comment (ps1312) [reply FeedAPIChallenge/Feed Feature/FeedImage.swift:7]: Thanks Mike! Must've left while doing some refactoring, but i do understand why FeedImage doesn't need to be Decodable!

Comment (mapostolakis) [reply FeedAPIChallenge/Feed Feature/FeedImage.swift:7]: Awesome, Paulo 👍
